.ci/static-checks.sh

```diff
@@ -42,6 +42,8 @@ specific_branch="false"
 force="false"
 branch=${branch:-main}
 
+user_agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
```

Comment:
Please make `user_agent` a local variable in `check_url()`.

My only worry about this solution is to eventually keep updating the version of the browser in the user agent string. However I don't have a better option to offer. I also tried with a string indicating the agent is a bot  ("Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)") but it didn't work out.

Reply:
Yeah, not sure. Given that their website shows up on Google search, they must be checking other things when checking against certain user agents.
Obviously cannot tell how often they update their filters or if its automatic by their provider.
Is there no case for just using a slash '/' as a user-agent? I guess the case against it is that its likely just an oversight that might get patched and may not exist on other websites that have user-agent filtering. But I don't know if there are any other sites that we care about at the moment, and it looks like at least for all Akamai-hosted sites I've seen that '/' works fine.

Reply:
@jodh-intel do you have any opinion here?